File: app/src/main/java/com/example/shopify/productinfo/model/pojo/Price.java

package com.example.shopify.productinfo.model.pojo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Price {

    @SerializedName("amount")
    @Expose
    private String amount;
    @SerializedName("currency_code")
    @Expose
    private String currencyCode;

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public String getCurrencyCode() {
        return currencyCode;
    }

    public void setCurrencyCode(String currencyCode) {
        this.currencyCode = currencyCode;
    }

}

File: app/src/main/java/com/example/shopify/productinfo/model/pojo/Price.kt
package com.example.shopify.productinfo.model.pojo

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

class Price {
    @SerializedName("amount")
    @Expose
    var amount: String? = null

    @SerializedName("currency_code")
    @Expose
    var currencyCode: String? = null
}